30-helm/multi-region/pingfederate/base.yaml

```diff
@@ -0,0 +1,69 @@
+global:
+  ingress:
+    enabled: true
+    addReleaseNameToHost: prepend
+    defaultDomain: ping-devops.com
+    defaultTlsSecret:
+    annotations:
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+      kubernetes.io/ingress.class: "nginx-public"
+
+pingfederate-admin:
+  enabled: true
+  container:
+    replicaCount: 1
+  envs:
+    SERVER_PROFILE_BASELINE_URL: https://github.com/pingidentity/pingidentity-server-profiles.git
+    SERVER_PROFILE_BASELINE_PATH: baseline/pingfederate
+    SERVER_PROFILE_PARENT: BASELINE
+    ## Using this profile layer for PF_NODE_GROUP_ID var
+    SERVER_PROFILE_URL: https://github.com/pingidentity/pingidentity-server-profiles.git
+    SERVER_PROFILE_PATH: pf-k8s-multi-clustering-layer/pingfederate
+  workload:
```

Comment:
we can leave off the statefulset. it's relevant to this example, but not required. 
it would just lead to more questions about PF operating patterns.